.github/workflows/ccpp.yml

```diff
@@ -46,7 +46,15 @@ jobs:
     - name: Build
       run: |
            cd gccrs-build; \
-           make -j $(nproc)
+           LC_ALL=C make -j $(nproc) > log 2>&1;
+           
+    - name: Build logs
+      run: cat gccrs-build/log
```

Comment:
May have to tune this: I suppose (but have not verified!) that in case that `make` fails, the whole `jobs.build-and-check` stops, and *Build logs* isn't executed, and thus there's no indication in the GitHub UI why `make` failed.

Reply:
Yes, that's true. I didn't notice :( 

Maybe we could do something like `LC_ALL=C make -j $(nproc) > log 2>&1 && cat gccrs-build/log` and then remove the separate task. If `make` fails the whole command should fail.

Reply:
Right, working on that, will push in a moment -- thanks for confirming!